### PR TITLE
Detangle the dependencies, particularly collection.

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?util=pull/86/head&deploy_to_bintray=false">this link.</A>

**Comment by eed3si9n** [comment]
What's the motivation here? Are you trying to move collection up to sbt/sbt?

**Comment by dwijnand** [comment]
That is correct.

File: build.sbt [hunk 1]
```diff
@@ -76,6 +76,11 @@ lazy val utilControl = (project in internalPath / "util-control").
     name := "Util Control"
   )
 
+val utilShowLines = (project in file("internal") / "util-showlines").settings(
```
Comment (eed3si9n) [inline build.sbt]: Can this move to logging then?

Comment (dwijnand) [reply build.sbt]: Sure.

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?util=pull/86/head&deploy_to_bintray=false">this link.</A>

**Comment by eed3si9n** [comment]
Ovrerall LGTM, but let me hold off on merging this until IO propagation is done.

**Comment by dwijnand** [comment]
Sure, np. I'll rebase it tomorrow.

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?util=pull/86/head&deploy_to_bintray=false">this link.</A>

**Pull request merged**